aruco_info[] aruco_list
